skype/skype-ps/skype/Set-CsOnlineLisCivicAddress.md

```diff
@@ -16,6 +16,8 @@ ms.reviewer:
 Use the \`Set-CsOnlineLisCivicAddress\` cmdlet to modify an existing civic address which has not been validated.
 Validated civic addresses cannot be modified.
 
+**Note:** This cmdlet is only available for Microsoft Internal use.
```

Comment:
```suggestion
**Note:** This parameter is for Microsoft internal use only.
```

Reply:
@JohanFreelancer9 it is the full cmdlet that is only available to Microsoft